src/custom/components/swap/UnsupportedCurrencyFooter/UnsupportedCurrencyFooterMod.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { ReactNode, useState } from 'react'
```

Comment:
I really like what you did in this MOD, you basically replaced the hardcoded texts by arbitrary components passed in the props